sycl/doc/extensions/experimental/sycl_ext_intel_esimd/README.md

```diff
@@ -61,14 +70,21 @@ To compile using Intel(R) OneAPI Toolkit:
 To run on an Intel GPU device through the Level Zero backend:
 > `$ SYCL_DEVICE_FILTER=level_zero:gpu ./a.out`
 
-The resulting executable (`a.out`) can only be run on Intel GPU hardware, such as
-Intel(R) UHD Graphics 600 or later. The DPC++ runtime automatically recognizes ESIMD
-kernels and dispatches their execution, so no additional setup is needed. Both Linux
-and Windows platforms are supported, including OpenCL and Level Zero backends.
+To run on an ESIMD_EMULATOR backend:
```

Comment:
please don't intermix Esimd emulator docs with GPU docs. I suggest to introduce a notion of modes (and later use them in the main emulator document too) - native and emulation, when execution is on GPU and CPU respectively. Then, not touching the original bullets, add a subsection for the emulator.